### PR TITLE
Solved the Issue #3452

### PR DESCRIPTION
**Pull request by SharmaDhruv2511**
### Feature Preview

 
This PR fixes the issue https://github.com/AppFlowy-IO/AppFlowy/issues/3452
---

#### PR Checklist

- [x] My code adheres to [AppFlowy's Conventions](https://docs.appflowy.io/docs/documentation/software-contributions/conventions)
- [x] I've listed at least one issue that this PR fixes in the description above.
- [x] I've added a test(s) to validate changes in this PR, or this PR only contains semantic changes.
- [ ] All existing tests are passing.

### CODE REVIEW

**Comment by MayurSMahajan** [comment]
@SharmaDhruv2511 use more descriptive titles in your future PRs 😅

**Comment by SharmaDhruv2511** [comment]
Yeah Sure. Can you run the test cases to merge the pull request @MayurSMahajan

**Comment by MayurSMahajan** [comment]
Actually I am not authorised to do so, but soon the maintainers will do it.

File: frontend/appflowy_flutter/lib/plugins/database_view/grid/presentation/widgets/header/type_option/number.dart [hunk 1]
```diff
@@ -73,6 +73,7 @@ class NumberTypeOptionWidget extends TypeOptionWidget {
             height: GridSize.popoverItemHeight,
             alignment: Alignment.centerLeft,
             child: FlowyText.medium(
+              color: Colors.grey,
```
Comment (Xazin) [inline frontend/appflowy_flutter/lib/plugins/database_view/grid/presentation/widgets/header/type_option/number.dart:76]: Use or add a color that is based on `AFThemeExtension` or `Theme.of(context).colorScheme`.

**Pull request merged**